float64[] pick_pose
float64[] place_pose